### ISSUE TITLE
File Uploads for Data Objects timing out

### ISSUE THREAD

**Issue by kellyjandrews** [Syncano/syncano-js#189]
Given the following code - 

```
var obj = {
  file: Syncano.file(fs.readFileSync('64kb-image.gif')),
  instanceName: 'sample-instance',
  className: 'files'
};

DataObject.please().create(obj)
  .then(function(object) {
    console.log('File: ', object);
  })
  .catch(function(err){
    console.log('err gif: ' + err);
  });
```

Results in `err gif: Error: timeout of 15000ms exceeded`

Directly posting the same file to the API endpoint is successful.  

`Syncano.file(fs.readFileSync('64kb-image.gif'))` returns what I believe is intended - `SyncanoFile {
  content: <Buffer 47 49 46 38 39 61 b7 01 a6 01 f7 ff 00 00 03 2f 77 c8 d9 c9 c9 c9 f5 f5 f5 e4 84 27 54 94 bb df 71 10 ff fd 6e 47 77 a9 ba ba ba ff f9 01 0c 81 bd fa ... > }` but something isn't quite making it across the api call.

**Comment by dancio** [comment]
just try to do this like this:
`Syncano.file(__dirname + '/64kb-image.gif')`

**Comment by kellyjandrews** [comment]
```
var obj = {
  file: Syncano.file(__dirname + '/64kb-image.gif'),
  instanceName: 'sample-instance',
  className: 'files'
};
```

Also produces the same error.

**Comment by jakubbilko** [comment]
Fixed in #220
